src/styles/utils/media.scss

```diff
@@ -0,0 +1,18 @@
+// media querys for cards block with different number of columns for  different width sreens
+@media (max-width: 487px) {
```

Comment:
no need to create a separate file for it - it should be inside .card-container:

```
.cards_container {
  display: grid;
  grid-template-columns: repeat(4, $card-width);
  gap: 48px;
  row-gap: 46px;
  justify-content: center;
  padding: 50px 40px;
  
  @media() {
    // styles
  }
}
```

Reply:
fixed